core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJResolver.scala

```diff
@@ -43,54 +43,40 @@ case class IntelliJPatternResolver(pattern: String) extends IntelliJResolver {
       "version" -> version.releaseOrBuild,
       "release" -> version.release.getOrElse("snapshot-release")
     )
-    val replacedBeforeResolvingGlobs = replacements.foldLeft(pattern) { case (path, (pattern, replacement)) =>
+    val artifactWithReplacementsApplied = replacements.foldLeft(pattern) { case (path, (pattern, replacement)) =>
       path.replace(s"[$pattern]", replacement)
     }
-    val replaced =
-      if (replacedBeforeResolvingGlobs.startsWith("file:"))
-        resolveGlobsInPattern(replacedBeforeResolvingGlobs)
-      else
-        replacedBeforeResolvingGlobs
 
-    Dependency(replaced)
+    if (artifactWithReplacementsApplied.startsWith("file:"))
+      findFilesMatchingGlob(artifactWithReplacementsApplied.stripPrefix("file:"))
+        .map("file:" + _)
```

Comment:
shouldn't we stript `file://`?

Reply:
Unfortunately, even the URIs returned by `java.lang.Class#getResource` (like, `java.lang.Class#getResource`) are only prefixed with `file:` instead of `file://`... so we probably shouldn't rely on standard being observed here 😅